### PR TITLE
Assessment Service - Caching + integration test coverage

### PR DESCRIPTION
**Pull request by burzum619**

### CODE REVIEW

File: service/src/main/java/tds/assessment/services/impl/AssessmentServiceImpl.java [hunk 1]
```diff
@@ -40,8 +42,9 @@ public AssessmentServiceImpl(AssessmentQueryRepository assessmentQueryRepository
     }
 
     @Override
+    @Cacheable(CacheType.LONG_TERM)
     public Optional<Assessment> findAssessment(final String clientName, final String assessmentKey) {
-        Optional<Assessment> maybeAssessment = assessmentQueryRepository.findAssessmentByKey(assessmentKey, clientName);
+        Optional<Assessment> maybeAssessment = assessmentQueryRepository.findAssessmentByKey(clientName, assessmentKey);
```
Comment (burzum619) [inline service/src/main/java/tds/assessment/services/impl/AssessmentServiceImpl.java:47]: Noticed this bug while I was writing my integration test - parameters were switched up here.

File: service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import tds.assessment.repositories.AssessmentQueryRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+public class AssessmentServiceImplIntegrationTest {
```
Comment (millerts) [inline service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplIntegrationTest.java]: Looking at the pom.xml integration tests must be plural in the name.  So this should be renamed `AssessmentServiceImplIntegrationTests`.  You will notice this doesn't execute via Maven without the change.

**Review by millerts — CHANGES_REQUESTED** [review]
One small change.

File: service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplIntegrationTest.java (hunk 2 above)
Comment (burzum619) [reply service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplIntegrationTest.java]: Got it, I'll change and keep that in mind.

**Pull request merged**